test/MINLPTests/run_minlptests.jl

```diff
@@ -98,7 +101,27 @@ CONFIG["Ipopt"] = Dict(
 #     "infeasible_point" => AmplNLWriter.MOI.UNKNOWN_RESULT_STATUS,
 # )
 
-@testset "$(name)" for name in ["Ipopt", "Bonmin", "Couenne"]
+import Uno_jll
+
+CONFIG["Uno"] = Dict(
+    "mixed-integer" => false,
+    "amplexe" => Uno_jll.amplexe,
+    "options" => String["preset=ipopt", "linear_solver=MUMPS"],
+    "tol" => 1e-5,
+    "dual_tol" => 1e-5,
+    "nlp_exclude" =>
+        ["005_010", "005_011", "006_010", "007_010", "008_010", "008_011"],
```

Comment:
When I ran locally, there were only a few failures. I'll see if this holds in CI.

Reply:
These are https://github.com/cvanaret/Uno/issues/38, https://github.com/cvanaret/Uno/issues/39, and https://github.com/cvanaret/Uno/issues/40.